README.md

```diff
@@ -126,7 +147,17 @@ time = pandas.date_range("2018-01-01", periods=3, freq="M")
 
 If you want to contribute to this project, please fork the repository or create a new branch containing your changes.
 
-**Install the pre-commit hooks** in our local git repo before commiting to ensure homogenous code style.
+**Install dependencies for development**
```

Comment:
sorry, again conda: do you want to mark which ones are not available there?

Reply:
yes, good idea. Will add it to the readme.